applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp

```diff
@@ -1876,10 +1876,10 @@ void UPwSmallStrainInterfaceElement<TDim, TNumNodes>::CalculateAndAddCouplingMat
 {
     KRATOS_TRY
 
-    Matrix b_matrix = prod(rVariables.RotationMatrix, rVariables.Nu);
+    Matrix like_b_matrix = prod(rVariables.RotationMatrix, rVariables.Nu);
```

Comment:
Had a brief discussion with @WPK4FEM and this matrix is still the b-matrix, it is just quite different for interface elements, so it doesn't fit in the stress state policy right now (we might change that in the future if we think it's feasible). Therefore, I'd suggest to revert this change (also I think the matrix can be probably const, missed that last time):
```suggestion
    const Matrix b_matrix = prod(rVariables.RotationMatrix, rVariables.Nu);
```

Reply:
Thank you very much for checking with WP. Reverted.